### PR TITLE
Remove conmon, get package from CRI-O

### PR DESCRIPTION
**Pull request by rhatdan**
conmon should not be built in two different places.
conmon is now a separate package in Fedora so we can just
add requires, for use on Ubuntu we can just require cri-o to
be installed.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
Can we add something in the README/tutorial to help folks building from source get a working copy?

File: Makefile [hunk 1]
```diff
@@ -100,7 +97,7 @@ endif
 	find . -name \*~ -delete
 	find . -name \#\* -delete
 	rm -f bin/podman
-	make -C conmon clean
+	make -C clean
```
Comment (mheon) [inline Makefile]: This should be removed - we don't have a conmon folder anymore, so no need to invoke make again on that folder

File: .papr.sh [hunk 2]
```diff
@@ -55,6 +55,7 @@ export PATH=$HOME/gopath/bin:$PATH
 export GOSRC=/$GOPATH/src/github.com/projectatomic/libpod
 
 ${PACKAGER} install -y \
```
Comment (lsm5) [inline .papr.sh]: might wanna install the [centos-release-container](http://cbs.centos.org/kojifiles/packages/centos-release-container/1/6.el7.centos/noarch/centos-release-container-1-6.el7.centos.noarch.rpm) package for CentOS envs before searching for cri-o rpm

File: .papr.sh [hunk 3]
```diff
@@ -55,6 +55,7 @@ export PATH=$HOME/gopath/bin:$PATH
...
+	      cri-o \
```
Comment (baude) [inline .papr.sh]: can we just install conmon ?

Comment (rhatdan) [reply .papr.sh]: I don't think this has worked its way out to RHEL and Centos yet.

**Comment by rhatdan** [comment]
Amazing this finally passes the tests.
@mheon @baude @umohnani8 @TomSweeneyRedHat PTAL

**Comment by mheon** [comment]
LGTM, though we should add instructions for building from source to the tutorial

**Comment by rhatdan** [comment]
Lets do that as a separate PR.

**Comment by TomSweeneyRedHat** [comment]
LGTM, and it's green, SHIP IT!

**Comment by umohnani8** [comment]
LGTM

**Comment by mheon** [comment]
@rh-atomic-bot retry

**Pull request closed without merging**